### PR TITLE
Add gcp-build samples

### PR DESCRIPTION
**Pull request by ace-n**
~~**Do not merge** - this is missing a (TypeScript) sample.~~

### CODE REVIEW

**Comment by steren** [comment]
Can you add a README.md explaining what the sample does and what is happening when `gcp-build` is run?

File: appengine/uglifyjs/static/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Simple web page</title>
+    <script src="message.js" type="text/javascript"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
```
[the remark this answers is not in the document]
Comment (ace-n) [reply appengine/uglifyjs/static/index.html]: I feel like this should be obvious based on the different page URLs (`/index` vs `/index_min`)

File: appengine/uglifyjs/static/index_min.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Simple web page</title>
+    <script src="message.min.js" type="text/javascript"></script>
+    <link rel="stylesheet" href="style.min.css">
+  </head>
+  <body>
```
[the remark this answers is not in the document]
Comment (ace-n) [reply appengine/uglifyjs/static/index_min.html]: Ditto to below comment.

File: appengine/typescript/index.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* tslint:disable:no-console no-var-requires */
+
+declare var require: any;
```
Comment (steren) [inline appengine/typescript/index.ts]: I think we can do better as a TS example.
This seems to be the copy paste of my little experiement, but does not feel very TS idiomatic: doe example, the Express types should be loaded from npm and I would expect the function to use typed objects.

File: appengine/typescript/app.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+runtime: nodejs8
+
+handlers:
```
Comment (steren) [inline appengine/typescript/app.yaml]: remove handlers, they are optional.

File: appengine/typescript/index.ts (hunk 3 above)
Comment (ace-n) [reply appengine/typescript/index.ts]: Yes, it is largely a copy-paste as I'm not terribly familiar with Typescript 😛

@JustinBeckwith is our resident TS expert, so I'll see what he suggests.

File: appengine/typescript/app.yaml (hunk 4 above)
Comment (ace-n) [reply appengine/typescript/app.yaml]: Done.

Comment (steren) [reply appengine/typescript/app.yaml]: Did you push the commit?

File: appengine/typescript/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## Setup
+
+Install dependencies:
+
+    With `npm`:
+
+        npm install
+
+    or with `yarn`:
```
Comment (fhinkel) [inline appengine/typescript/README.md]: Personal preference, I would not include `yarn` here. Our Node docs say that you can use `npm` or `yarn`, no need to show everything twice here.

Comment (ace-n) [reply appengine/typescript/README.md]: Removed `yarn` for brevity.

**Comment by fhinkel** [comment]
Thanks for adding tests, can you add them to kokoro, too?

**Comment by ace-n** [comment]
Done - do we need to merge this first?

**Pull request merged**